spec/models/repository_host/gitlab_spec.rb

```diff
@@ -3,6 +3,14 @@
 require "rails_helper"
 
 describe RepositoryHost::Gitlab do
-  let(:repository) { build(:repository, host_type: "GitLab") }
+  let(:repository) { build(:repository, host_type: "GitLab", full_name: "ase/ase") }
   let(:repository_host) { described_class.new(repository) }
+  let(:api_token) { "TEST_TOKEN" }
+
+  it "can fetch repository data" do
+    VCR.use_cassette("gitlab/ase") do
+      repository_data = described_class.fetch_repo(repository.id_or_name, api_token)
+      expect(repository_data).not_to be_nil
```

Comment:
Should these spex do more? e.g. expect the repository_data object to have successfully extracted specific elements / keys with the correct values, and that it's not just getting all the keys and populating them with nils or data from the wrong fields?